src/configs/env.ts

```diff
@@ -201,3 +201,9 @@ export type DEFAULT_PLAN_VERSION = typeof DEFAULT_PLAN_VERSION;
 
 export const ENABLE_PRACTITIONERS = process.env.REACT_APP_ENABLE_PRACTITIONERS === 'true';
 export type ENABLE_PRACTITIONERS = typeof ENABLE_PRACTITIONERS;
+
+export const PRACTITIONER_FORM_NAMESPACE = '78295ac0-df73-11e9-a54b-dbf5e5b2d668';
```

Comment:
Can we read this from an env variables, having this value as the fallback default? e.g. https://github.com/onaio/reveal-frontend/blob/master/src/configs/env.ts#L8